Classes/Events/PlaylistEvent.swift

```diff
@@ -0,0 +1,72 @@
+// ===================================================================================================
+// Copyright (C) 2021 Kaltura Inc.
+//
+// Licensed under the AGPLv3 license, unless a different license for a
+// particular library is specified in the applicable library path.
+//
+// You may obtain a copy of the License at
+// https://www.gnu.org/licenses/agpl-3.0.html
+// ===================================================================================================
+//
+//  PlaylistEvent.swift
+//  PlayKit
+//
+//  Created by Sergii Chausov on 27.10.2021.
+//
+
+import Foundation
+
+@objc public class PlaylistEvent: PKEvent {
+    
+    @objc public static let allEventTypes: [PlaylistEvent.Type] = [
+        playlistLoaded, playlistStarted, playlistEnded, playlistCountDownStart, playlistCountDownEnd, playlistLoopStateChanged, playlistLoopStateChanged, playlistAutoContinueStateChanged, playlistError, playlistLoadMediaError, playlistCurrentPlayingItemChanged
+    ]
+    
+    @objc public static let playlistLoaded: PlaylistEvent.Type = PlaylistLoaded.self
+    @objc public static let playlistStarted: PlaylistEvent.Type = PlaylistStarted.self // ?
+    @objc public static let playlistEnded: PlaylistEvent.Type = PlaylistEnded.self
+    @objc public static let playlistCountDownStart: PlaylistEvent.Type = PlaylistCountDownStart.self // ?
+    @objc public static let playlistCountDownEnd: PlaylistEvent.Type = PlaylistCountDownEnd.self // ?
+    @objc public static let playlistLoopStateChanged: PlaylistEvent.Type = PlaylistLoopStateChanged.self
+    @objc public static let playlistAutoContinueStateChanged: PlaylistEvent.Type = PlaylistAutoContinueStateChanged.self
+    @objc public static let playlistError: PlaylistEvent.Type = PlaylistError.self
+    @objc public static let playlistLoadMediaError: PlaylistEvent.Type = PlaylistLoadMediaError.self
+    @objc public static let playlistCurrentPlayingItemChanged: PlaylistEvent.Type = PlaylistCurrentPlayingItemChanged.self
+    
+    public class PlaylistLoaded: PlaylistEvent {}
```

Comment:
@x-NR-x should we have any payload for: PlaylistLoaded, PlaylistStarted, PlaylistEnded, PlaylistCountDownStart, PlaylistCountDownEnd, PlaylistLoopStateChanged, PlaylistAutoContinueStateChanged ?